File: DBBasic/output_obj/getpass_obj.cs
using DBBasic.Model;
using System;
using System.Collections.Generic;
using System.Text;

namespace DBBasic.output_obj
{
    public class getpass_obj : Serializable
    {
        public List<passi_expres> passis;
        public int num_passis;
        public getpass_obj(List<passi_expres> passis)
        {
            this.passis = passis;
        }
        public new byte[] serialize(bool header = true)
        {
            num_passis = passis.Count;
            return base.serialize();
        }
    }
}

File: Socis/app/src/main/java/com/app/socis/input_obj/getpass_obj.java
package com.app.socis.input_obj;

import com.app.socis.model.passi_expres;

import java.util.List;

public class getpass_obj {
	public List<passi_expres> passis;
	public int num_passis;

	public getpass_obj(List<passi_expres> passis, int num_passis) {
		this.passis = passis;
		this.num_passis = num_passis;
	}
}
